packages/api-client-next/src/transformPathToQuery.ts

```diff
@@ -68,7 +70,13 @@ export function transformPathToQuery<T extends Record<string, unknown>>(
       returnOptions.body ??= {} as T;
       Reflect.set(returnOptions.body, key, params[key]);
     }
-  });
+  }
+
+  // Exception for unknown type of FormData - multipart/form-data MIME type
+  // pass as it is
+  if (params instanceof FormData) {
+    returnOptions.body ??= params as T;
+  }
```

Comment:
we need to add tests for that ad I'm not sure in which cases it should be considered

Reply:
it's the only case when we pass a FormData (multipart/form-data content type), it cannot be split into smaller pieces or destructured. so it's always a one compact object. 
I added a test-case for that.